### ISSUE TITLE
Minor: Accessibility (edited)

### ISSUE THREAD

**Issue by rsgoertz** [info442-group-7/HelpOut#39]
Accessibility:

- Screen reader can read simple text but has no designations for buttons, links, homepages, action buttons (dismiss, info, confirm) on task cards. 

This may be due to a large number of missing or incorrect ARIA tags. 

This presents a serious accessibility/usability issue for visually impaired users that rely on screen readers to navigate and derive functionality from websites. In its current state, HelpOut cannot be used by these users.

**Comment by pranita122** [comment]
Thanks for the feedback! We are going to mark this as out of the scope of our requirements. It's a great suggestion, but not one we plan on implementing for this release. Additionally, as it doesn't break functionality, we are going to downgrade this as a minor bug.
